wagtail_airtable/management/commands/import_airtable.py

```diff
@@ -660,13 +25,20 @@ def handle(self, *args, **options):
             if AIRTABLE_DEBUG:
                 options["verbosity"] = 2
 
-        importer = Importer(models=options["labels"], options=options)
-        created, skipped, updated = importer.run()
+        model = get_validated_models([options["model_name"]])[0]
```

Comment:
Looks like this is losing the ability to pass multiple models on the command line - ensure that we reinstate this.